src/layout/TopBar/TopBar.js

```diff
@@ -55,12 +91,33 @@ function TopBar({
         </Link>
 
         <div className={classes.menuRight}>
-          <Link to={routes.USER_MANAGEMENT}>
-            <IconButton aria-label="user-management" color="inherit">
-              <GroupIcon fontSize="large" className={classes.menuIcon} />
-            </IconButton>
-          </Link>
-          <IconButton aria-label="logout" color="inherit" onClick={handleLogoutClick}>
+          {isSuperAdmin && (
```

Comment:
Check the placement as well. It is all disturbed.

![image](https://user-images.githubusercontent.com/70138493/159242836-3d4e436b-f132-48d8-89d1-e2d51da24f53.png)

Reply:
Radhika can you tell me screen resolution, for me all medium and large screens placement is proper

Reply:
1320 x 1000

Reply:
![image](https://user-images.githubusercontent.com/60848892/159471009-2928bc2b-5728-41a6-8a8f-1177921d4cfd.png)